IPython/core/historyapp.py

```diff
@@ -30,6 +36,123 @@
 """
 
 
+class HistoryShow(BaseIPythonApplication):
+    description = show_hist_help
+    limit = Int(1000, help="Number of recent history entries to show.").tag(config=True)
+
+    delimiter = Unicode(
+        "\t", help="Delimiter for separating the columns of history output."
+    ).tag(config=True)
+
+    header = Bool(False, help="Include a header row in the output.").tag(config=True)
+
+    pretty = Bool(
+        False, help="Pretty-print the output using IPython.utils.PyColorize."
+    ).tag(config=True)
+
+    colors = CaselessStrEnum(
+        ("Neutral", "NoColor", "LightBG", "Linux"),
+        default_value="Neutral",
+        help="Set the color scheme (NoColor, Neutral, Linux, or LightBG).",
+    ).tag(config=True)
```

Comment:
This have changed in 9.0, now we have themes, that are arbitrary strings, the Neutral, CoColor, LightBG, and Linux still exists but are lower case.